pkg/helm/cluster.go

```diff
@@ -87,3 +89,35 @@ func (h *Helm) OverrideDesiredVersion(rls *output.ReleaseOutput) {
 		}
 	}
 }
+
+// filterIgnoredReleases is a helper function that removes charts that match a release name or chart name
+// provided by the user at runtime from the list of found charts in the cluster
+func filterIgnoredReleases(deployed []*release.Release, releaseIgnoreList []string, chartIgnoreList []string) []*release.Release {
+	// Filter out any ignored releases
+	filteredDeployed := make([]*release.Release, len(deployed))
+	filteredCount := 0
+
+	for _, release := range deployed {
+		isIgnoredRelease := false
+		isIgnoredChart := false
+		for _, ignoreListedRelease := range releaseIgnoreList {
+			if release.Name == ignoreListedRelease {
+				isIgnoredRelease = true
+			}
+		}
+		for _, ignoreListedChart := range chartIgnoreList {
+			// Check for nil to avoid a potential nil pointer exception
+			if release.Chart != nil {
+				if release.Chart.Name() == ignoreListedChart {
+					isIgnoredChart = true
+				}
+			}
+		}
+		if !isIgnoredChart && !isIgnoredRelease {
+			filteredDeployed[filteredCount] = release
+			filteredCount++
```

Comment:
Rather than tracking the count separately, it'd probably simplify things to just use `append`, even though you don't get the bonus of pre-allocating the slice

```suggestion
			filteredDeployed = append(filteredDeployed, release)
```